CMakeLists.txt

```diff
@@ -20,20 +21,10 @@ message(STATUS "Debug mode ON" )
 else(DEBUG STREQUAL "ON")
 # Release mode
 set(CMAKE_BUILD_TYPE Release)
-set(CMAKE_C_FLAGS "-O2")
```

Comment:
This should be set by host/distro. In case it's really needed, it should have CACHE keyword to be overridable by distro CFLAGS.